### PR TITLE
Docker setup

### PR DESCRIPTION
**Pull request by educatedwarrior**

### CODE REVIEW

**Comment by educatedwarrior** [comment]
pull recent changes

File: Docker/entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+	    cp /genesis-test.json ${DATADIR}
+	    exec mused -s ${TEST_SEED} \
+		--replay-blockchain --rpc-endpoint=0.0.0.0:8090 \
+		--genesis-json ${DATADIR}/genesis-test.json \
+		-d ${DATADIR}/
+	fi
+
+fi
+
+if [ "${NODE_TYPE}" = "prod" ]; then
```
Comment (ghost) [inline Docker/entrypoint.sh:32]: For production, genesis json is embedded, see egenesis_full.cpp
Using genesis.json is not required.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+              \"optional\": true                \
+          }                            \
+      ]"
+
+#Build blockchain source
+RUN \
+	cd /tmp && git clone https://github.com/themuseblockchain/Muse-Source.git && \
+	cd Muse-Source && \
+	git submodule update --init --recursive && \
+	cmake -j 8 -DBOOST_ROOT="$BOOST_ROOT" -DBUILD_MUSE_TEST=OFF -DCMAKE_BUILD_TYPE=Debug . && \
```
Comment (ghost) [inline Dockerfile]: Use -DCMAKE_BUILD_TYPE=Release instead?

File: Docker/entrypoint.sh (hunk 1 above)
Comment (tilllinden) [reply Docker/entrypoint.sh:32]: right!

File: Dockerfile (hunk 2 above)
Comment (tilllinden) [reply Dockerfile]: Can be both, if you wanna attach gdb/lldb for any reason debug is required.

**Comment by Tadasu85** [comment]
Nice work EW! When are we merging this guys?

File: Docker/entrypoint.sh (hunk 1 above)
Comment (educatedwarrior) [reply Docker/entrypoint.sh:32]: I'll remove.

**Pull request merged**